### PR TITLE
Add default log parameters

### PR DESCRIPTION
**Pull request by Mart124**
Hi,

This PR adds some new default log parameters.
This solves #668 for the `logRollBySizeLimit` parameter.
And completes #742 for the `Syslog.Destination` one.

Thank you 👍

### CODE REVIEW

File: util/Setup/EnvironmentFileBuilder.cs [hunk 1]
```diff
@@ -95,6 +95,8 @@ private void Init()
                 ["globalSettings__attachment__baseUrl"] = $"{_context.Config.Url}/attachments",
                 ["globalSettings__dataProtection__directory"] = $"{_context.OutputDir}/core/aspnet-dataprotection",
                 ["globalSettings__logDirectory"] = $"{_context.OutputDir}/logs",
+                ["globalSettings__logRollBySizeLimit"] = string.Empty,
+                ["globalSettings__Syslog__Destination"] = string.Empty,
```
Comment (cscharf) [inline util/Setup/EnvironmentFileBuilder.cs:99]: Following convention should use camelCase vs. PascalCasing, i.e. `["globalSettings__syslog__destination"]`

Comment (Mart124) [reply util/Setup/EnvironmentFileBuilder.cs:99]: I agree, but #742 uses `Syslog.Destination` in its code...

Comment (cscharf) [reply util/Setup/EnvironmentFileBuilder.cs:99]: Yeah, I'm not entirely sure how that mapping works, but `globalSettings.LogRollBySizeLimit` is also in the code, and the casing just "works".

Comment (Mart124) [reply util/Setup/EnvironmentFileBuilder.cs]: Case just updated then @cscharf, thank you for your review 👍

**Pull request merged**